bika/lims/controlpanel/bika_analysisservices.py

```diff
@@ -72,6 +72,9 @@ def copy_service(self, src_uid, dst_title, dst_keyword):
                     continue
                 value = field.get(src_service)
                 if value:
+                    # https://github.com/bikalabs/bika.lims/issues/2015
+                    if fieldname in ["UpperDetectionLimit", "LowerDetectionLimit"]:
+                        value = str(value)
```

Comment:
This does not belong to this Pull Request, rather to https://github.com/senaite/bika.lims/pull/294